src/ResourceManager/Sql/Commands.Sql/ManagedDatabase/Cmdlet/RestoreAzureSqlManagedDatabase.cs

```diff
@@ -127,6 +127,20 @@ public class RestoreAzureRmSqlManagedDatabase
             HelpMessage = "The name of the target managed database to restore to.")]
         public string TargetManagedDatabaseName { get; set; }
 
+        /// <summary>
+        /// Gets or sets the name of the target managed instance to restore to. If not specified, the target instance is same as source
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The name of the target managed instance to restore to.")]
+        public string TargetManagedInstanceName { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the target resource group to restore to. If not specified, the target resource group is same as source
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The name of the target resource group to restore to.")]
```

Comment:
Please document behavior if unspecified, e.g. "The name of the target resource group to restore to. If not specified, the target resource group is the same as the source resource group."